backstage-helm/values.yaml

```diff
@@ -76,7 +77,7 @@ upstream:
                 frequency: { minutes: 1 }
                 timeout: { minutes: 1 }
                 initialDelay: { seconds: 15 }
-          github:
+          bitbucketServer:
             systems:
```

Comment:
Close but the configuration is a bit different for `bitbucketServer` vs `github`.
Is should be this:
```
bitbucketServer:
      systems:
        host: 'bitbucket-bitbucket{{.Values.global.clusterRouterBase }}'
        catalogPath: '/systems/*.yaml'
        schedule:
          frequency: { minutes: 1 }
          initialDelay: { seconds: 15 }
          timeout: { minutes: 1 }
      components:
        host: 'bitbucket-bitbucket{{.Values.global.clusterRouterBase }}'
        catalogPath: '/catalog-info.yaml'
        schedule:
          frequency: { minutes: 1 }
          initialDelay: { seconds: 15 }
          timeout: { minutes: 1 }
      apis:
        host: 'bitbucket-bitbucket{{.Values.global.clusterRouterBase }}'
        catalogPath: '/catalog-info.yaml'
        schedule:
          frequency: { minutes: 1 }
          initialDelay: { seconds: 15 }
          timeout: { minutes: 1 }
 ```

Reply:
done